v2/cn/download.md.tt2

```diff
@@ -28,7 +28,7 @@ OpenResty<sup>&reg;</sup> 给常见 Linux 发布提供[官方预编译安装包]
 强烈建议 Mac OS X 或者 macOS 系统用户通过 [homebrew](https://brew.sh/) 包管理器安装 OpenResty，像下面这样：
 
 ```
-brew install homebrew/nginx/openresty
+brew install denji/nginx/openresty
```

Comment:
We don't want to use the `denji` namespace (nor `nginx`) in our official openresty homebrew.